src/Components/Product.js

```diff
@@ -1,22 +1,50 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { FetchProduct } from '../redux/API/productfetch';
+import { fetchProduct } from '../redux/API/productfetch';
+import Item from './Item';
 
-const ProductLists = () => {
+const Products = () => {
   const dispatch = useDispatch();
-  const Products = useSelector((state) => state.Products);
+  const products = useSelector((state) => state.products);
   useEffect(() => {
-    if (!Products.length) {
-      dispatch(FetchProduct);
-    }
+    dispatch(fetchProduct());
   }, []);
-  return (
-
-    <div>
-      <div>Get list of Products</div>
 
-    </div>
+  const [searchTerm, setSearchTerm] = useState('');
+  const filteredproducts = products.filter((product) => product.title
+    .toLowerCase().includes(searchTerm.toLowerCase()));
+  const handleChange = (e) => {
+    const { value } = e.target;
+    setSearchTerm(value);
+  };
+  return (
+    <>
+      <div className="welcome">Welcome Home Page</div>
+      <input
+        type="text"
+        name="search"
+        className="search"
+        placeholder="Search Item"
+        value={searchTerm}
+        onChange={handleChange}
+      />
+      <div className="products">
+        {filteredproducts.map((product) => (
+          <Item
+            key={product.id}
+            id={product.id}
+            title={product.title}
+            price={product.price}
+            description={product.description}
+            category={product.category}
+            image={product.image}
+            rate={product.rating.rate}
+            count={product.rating.count}
+          />
+        ))}
+      </div>
```

Comment:
- [ ] In addition, your design doesn't follow the design given which you can find[ here ](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) and that means there is still some work to do on your design. Please take a look at the comparison below 👇 

**Design given**
![image](https://user-images.githubusercontent.com/90222110/189452144-977734ea-6b1a-4ca1-ba2f-e3bb9191a9a6.png)
⚠️ NOTE that I have shown you the blue version of it but you have the freedom to choose among the two color versions available (blue and pink)

**Your Design**
![image](https://user-images.githubusercontent.com/90222110/189452354-a5429fb7-2ba1-48b6-a9d6-53db87d65613.png)

As you can see there is a very noticeable difference in the Layout (composition), typography (fonts, font size, and font-weight), and the colors
Please improve your design while sticking to the design given as much as possible 😃 👍